tests/plugins/autogrow/autogrow.js

```diff
@@ -8,7 +8,58 @@
 
 	bender.editor = {};
 
+	bender.createEditor = function( width, paragraphs ) {
+		bender.editorBot.create( {
+			name: 'EditorWidth' + width,
+			config: {
+				width: width
+			}
+		}, function( bot ) {
+			bot.setData( autogrowTools.getTestContent( paragraphs ), function() {
+				var editor = bot.editor;
+				var initialEditorSize = autogrowTools.getEditorSize( editor );
+
+				editor.once( 'afterCommandExec', function() {
+					resume( function name( params ) {
```

Comment:
`params` is not used, so could be omitted. We could skip`name` name also since it's not telling anything and we are using anonymous functions in most cases.